app/uk/gov/hmrc/eoricommoncomponent/frontend/views/helpers/inputRadioGroup.scala.html

```diff
@@ -75,8 +75,7 @@
             content = Text(legend),
             classes = legendClasses,
             isPageHeading = isLegendH1
-        )),
-        describedBy = Some(describedBy)
+        ))
```

Comment:
We should be removing the describedBy value declaration on the top of the page as well

Reply:
Fixed